packages/mc-scripts/src/commands/login.js

```diff
@@ -0,0 +1,52 @@
+const { promisify } = require('util');
+const read = promisify(require('read'));
+const { processConfig } = require('@commercetools-frontend/application-config');
+const CredentialsStorage = require('../utils/credentials-storage');
+const authenticator = require('../utils/auth');
+const validateEmail = require('../utils/validate-email');
+
+const getEmail = async () => {
+  let email = await read({ prompt: 'Email: ' });
+  if (!validateEmail(email)) {
+    console.log('Please input a valid email');
+    email = await getEmail();
+  }
+  return email;
+};
+
+const getPassword = async () => {
+  return await read({ prompt: 'Password: ', silent: true });
+};
+
+const successfulLoginMessage = (cloudIdentifier) => {
```

Comment:
I think we need to provide different messages. Therefore I would remove this function and just inline the logs when needed (see other comments).